### PR TITLE
Release systemd RPMs to github releases

### PR DESCRIPTION
**Pull request by astj**
- Make `make rpm{,-stage,-kcps}` to build systemd RPMs too
- Update changelog in systemd RPM in release process (same as original RPM)
- Some refactor

# Discussion

Currently mackerel-agent-systemd.spec has no changelogs before 0.40.1.  Should we add them?

### CODE REVIEW

File: _tools/releng [hunk 1]
```diff
@@ -542,6 +547,10 @@ sub main {
         infof "Upload files to GitHub Release.\n";
         upload_to_github_release();
         return;
+    } elsif($task && $task eq "update-changelog"){
```
Comment (astj) [inline _tools/releng]: This is for debugging.

File: Makefile [hunk 2]
```diff
@@ -141,4 +152,4 @@ clean:
 
 generate: commands_gen.go
 
-.PHONY: test build run deps clean lint crossbuild cover rpm deb tgz generate crossbuild-package crossbuild-package-kcps crossbuild-package-stage rpm-systemd
+.PHONY: test build run deps clean lint crossbuild cover rpm deb tgz generate crossbuild-package crossbuild-package-kcps crossbuild-package-stage rpm-v1 rpm-v2 rpm-stage rpm-stage-v1 rpm-stage-v2 rpm-kcps-v1 rpm-kcps-v2
```
Comment (astj) [inline Makefile]: Some other make tasks don't exist in .PHONY……

File: .travis.yml [hunk 3]
```diff
@@ -22,6 +22,9 @@ script:
 - make crossbuild
 after_script:
 - goveralls -coverprofile=.profile.cov
+- docker pull astj/mackerel-rpm-builder:c5
+- docker pull astj/mackerel-rpm-builder:c7
+- make rpm rpm-kcps rpm-stage
```
Comment (astj) [inline .travis.yml]: I'll revert these changes in `.travis.yml` later. (I changed them just to check `make` works fine in travis.)

**Comment by astj** [comment]
Since ci of https://github.com/mackerelio/mackerel-agent/pull/347/commits/660cf560d984c90906124c0842ba2ed94c9de7ba passed, those make tasks seems to work fine.

**Comment by itchyny** [comment]
LGTM

**Comment by astj** [comment]
Thanks!

**Pull request merged**